src/components/Cube.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+import { CIRCLE_VALUE, CROSS_VALUE } from '../constants/game';
+
+const CIRCLE_SRC = '/static/circle.png';
+const CROSS_SRC = '/static/cross.png';
+const NEUTRAL_SRC = 'static/neutral.png';
+
+// for tests purposes
+const getTestId = ({ x, y, value, isMovable, isWinning }) => `cube-${x}-${y}-${value}-${isMovable}-${isWinning}`;
+
+const getLogo = value =>
+    ({
+        [CIRCLE_VALUE]: CIRCLE_SRC,
+        [CROSS_VALUE]: CROSS_SRC,
+    }[value] || NEUTRAL_SRC);
+
+const Cube = props => {
+    const testId = getTestId(props);
+    const { isMovable, handlePressCube, isSelected, isWinning, value } = props;
+    const logo = getLogo(value);
+    const image = !isSelected ? <CubeImage src={logo} /> : null;
+
+    const handlePress = isMovable ? handlePressCube : () => {};
```

Comment:
Instead of creating an empty function, `<StyledCube>` should accept `null` and do nothing IMO